ipfix-flow-exporter/bpf_ipfix_kern_common.h

```diff
@@ -32,6 +32,18 @@ typedef struct flow_record_ {
     u16 counter;                    /* flow_idle_counter */
 } flow_record_t;
 
+void *memset(void *b, int c, unsigned long len)
```

Comment:
Why are we adding this? I don't see a reference to it

Reply:
This Function is Present in <string.h> which we are not including hence I have implemented this function, when I am including <string.h> in our bpf Program it is creating lot of compilation issues.